src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  --outline: 0;
+  --border: 1px solid #e1e7ed;
+  --border-radius: 4px;
+  --main-text-color: #3d4e61;
+  --font-family: "Avenir", sans-serif;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.container {
+  margin-top: 20px;
+}
+
+.form {
+  position: relative;
+  height: 70px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px #3d4e6133;
+  border-radius: var(--border-radius);
+}
+
+.image {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  cursor: text;
+}
+
+.search {
+  outline: var(--outline);
+  padding-left: 62px;
+  border: var(--border);
```

Comment:
Consider using a more descriptive variable name for border. For example, `--search-border`. 🤔